### PR TITLE
TNT-38928 - implement CI/CD via github actions

### PR DESCRIPTION
**Pull request by dcottingham**
Implement CI/CD via build.yml and publish.yml github workflow files

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    branches: [main]
+
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        python-version: [2.7, 3.4, 3.5, 3.6, 3.7, 3.8, 3.9]
```
Comment (5amfung) [inline .github/workflows/build.yml:16]: Python 2.7 is no longer supported.  I think it's fine that we do the same.

File: .github/workflows/publish.yml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+      env:
+        TWINE_USERNAME: ${{ secrets.PYPI_USERNAME }}
+        TWINE_PASSWORD: ${{ secrets.PYPI_PASSWORD }}
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        GH_TOKEN: ${{ secrets.ADOBE_BOT_GITHUB_TOKEN }}
+        CI: true
+      run: |
+        make version_bump part=${{ github.event.inputs.bump }}
+        make build
+        make publish
```
Comment (5amfung) [inline .github/workflows/publish.yml:36]: Need a newline for EOF.

**Review by 5amfung — APPROVED** [review]
LGTM.  Just one nitpick on missing EOF newline.

**Review by mptap — APPROVED** [review]
Nice! 👍

**Pull request merged**